src/meson.build

```diff
@@ -64,10 +64,12 @@ test(
     '--target-dir=@0@'.format(cargo_target_dir),
     '--',
     '--nocapture',
+    '--test-threads=1',
```

Comment:
Is this still required, even though gtk tests already run single threaded?

Reply:
I wouldn't add it if it wasn't needed.

Reply:
I'm still not sure why this is still needed